modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/init.es.js

```diff
@@ -33,7 +33,7 @@ var initSPA = function(callback) {
 			{
 				handler: RenderURLScreen,
 				path: function(url) {
-					if (url.indexOf(themeDisplay.getPathMain()) === 0) {
+					if ((url + "/").indexOf(themeDisplay.getPathMain() + "/") === 0) {
 						return false;
```

Comment:
Hey @ricardocousolr, thanks for sending the fix! One question:

Wouldn´t 

```if (url.indexOf(themeDisplay.getPathMain() + '/') === 0) {
	return false;
}
````

Be enough?

Reply:
I think it wouldn't work well with the case in which the variable url is just "en/c".